src/components/RuntimeManager.js

```diff
@@ -199,10 +184,16 @@ export default class RuntimeManager extends PureComponent {
       })
       errorNotifiedApps.update(Utils.append(galaxyApp.appName))
     }
-  }
+  }, [runtimes, apps]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
What are the other things the effect depends on?

Reply:
It depends on `prevRuntimes`, `prevApps` which are computed off the included dependencies, and react is not able to figure it out. It also depends on `namespace, name` technically. Currently, the runtime is already scoped by these variables so its trivial to include them as well here since the runtime will refresh if they do. Can't hurt include these though in case that changes at some point.